Github/service.js

```diff
@@ -71,20 +71,23 @@ const getCommits = commitObject => {
           reject(err);
         } else {
           JSON.parse(data).forEach(singleCommit => {
-            const { commit: { author } } = singleCommit;
-
+            const {
+              commit: { author },
+            } = singleCommit;
             if (today - new Date(author.date) < recency) {
               const time = new Date(author.date);
-              singleCommit.author = {
-                name: author.name,
-                date: time.toLocaleString('en-US', {
-                  timeZone: 'America/Toronto',
-                }),
+              const simplifiedCommit = {
+                author: {
+                  name: author.name,
+                  date: time.toLocaleString('en-US', {
+                    timeZone: 'America/Toronto',
+                  }),
+                },
+                message: singleCommit.commit.message,
+                repoName: commitObject.repo,
+                branchName: commitObject.br.name,
               };
-              singleCommit.message = singleCommit.commit.message;
-              singleCommit.repoName = commitObject.repo;
-              singleCommit.branchName = commitObject.br.name;
-              recentCommits.push(singleCommit);
+              recentCommits.push(simplifiedCommit);
```

Comment:
`recentCommits` and `branchUrls` variables are quite hard to keep track of since these are defined at the beginning of the file & are modified by the exported functions. For example, it is hard to guess if `getAllCommitsTogether` requires `branchUrls` already populated unless you spend some time looking at the source code.

I am pretty sure we could chain them with `reslove(branchUrls)` statements instead & pass to the next service function as an argument.